optimum/exporters/onnx/model_configs.py

```diff
@@ -742,6 +744,94 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
+class HubertOnnxConfig(AudioOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedConfig
+
+
+class Wav2Vec2OnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class Wav2Vec2ConformerOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class SEWOnnxConfig(HubertOnnxConfig):
+    pass
+
+
+class SEWDOnnxConfig(HubertOnnxConfig):
+    DEFAULT_ONNX_OPSET = 12
```

Comment:
curiosity: why?

Reply:
The min (or clip) operator do not support in64 type before this opset.